### ISSUE TITLE
Abnormal model training

### ISSUE THREAD

**Issue by lzq04** [Teoge/DMPR-PS#13]
![image](https://user-images.githubusercontent.com/69559066/145709951-8c7d5265-74c5-48fd-aa7b-d2de75c4ca22.png)

Follow your approach to PS2 0 data set, the training set and test set obtained, and the weight file obtained after training are used for parking point detection and evaluation. The evaluation data of parking point detection is shown in the figure below:
![image](https://user-images.githubusercontent.com/69559066/145710009-14d1a360-b03e-48ed-93da-53d901ae342b.png)

In addition, the same data set is tested, and the weight file is modified to the author's. The evaluation data are as follows:
![image](https://user-images.githubusercontent.com/69559066/145710043-5d5c0b35-ae08-4d8e-9cce-f48e6f998975.png)

The effect given in the original paper is

![image](https://user-images.githubusercontent.com/69559066/145710082-ac89d957-7586-46cf-9dc4-17d4e9cc00ec.png)
So you need to explain to me why this happens? Or is there something wrong with my experiment?

**Comment by lzq04** [comment]
The environment I use is pytorch1 7 and CUDA 10.2

**Comment by Teoge** [comment]
Average precision is a metric different from precision and recall.
Take a look at [https://en.wikipedia.org/wiki/Evaluation_measures_(information_retrieval)#Average_precision](https://en.wikipedia.org/wiki/Evaluation_measures_(information_retrieval)#Average_precision) for details.
If you wish to reproduce the precision recall in the paper, please refer to the step in [here](https://github.com/Teoge/DMPR-PS/issues/6#issuecomment-751423126).
